### ISSUE TITLE
hoverNode and blurNode not called while stabilizing

### ISSUE THREAD

**Issue by controversial** [almende/vis#1770]
If the network is moving while stabilizing, and a node moves under the user's mouse cursor, the hoverNode event is not called until the network finishes stabilizing. Likewise, if a node that had been under the user's mouse cursor moves out from under the cursor during stabilization, the blurNode event is not called until the network stops moving.

Is this by design, or a bug?

**Comment by AlexDM0** [comment]
We listen to mouse movement so I guess that's the reason. It's not by design so technically it's a bug. If you could do a pull to fix it it would be a great help!

**Comment by The-Penultimate-Defenestrator** [comment]
I probably can't my skills are pretty limited. I'll take a look, though.

**Comment by AlexDM0** [comment]
If I were to guess, have the method that checks the hover state subscribe to the initRedraw. We should check performance though. 

Regards

**Comment by The-Penultimate-Defenestrator** [comment]
Yeah, I was thinking earlier that performance would be the most likely reason this wasn't implemented. We can see if I implement it.

**Comment by mojoaxel** [comment]
I'm closing this issue due to inactivity. 
If you think this issue is still relevant please feel free to reopen!

**Comment by controversial** [comment]
This is still relevant
